src/commonmark/en/content/user/additional-data-dimensions.md

```diff
@@ -46,4 +46,171 @@ organisation unit and period dimensions.
 ## Create or edit an attribute category and its options
 
 For the process of creating an attribute category as well as options and
-combinations, see the section of this manual named Manage categories.
+combinations, see **Chapter 21.2: Manage categories**. As discussed in that
+chapter, disaggregation categories are configured by editing a data
+element and attribute categories are configured by editing a data set.
+
+## Data entry with disaggregation categories and attribute categories
+
+When entering aggregate data, one must first choose the attribute
+categories, and then one can enter the data across disaggregation
+categories on a single page.
+
+For instance, in the graphic below, the attribute categories are
+Implementing Partner (AIDSRelief Consortium) and Project (Improve access
+to medicines).  The disaggregation categories are gender
+(male/female/etc.), age (<15, 15-24, 25-49, >49).
+
+![](resources/images/maintainence/categories_dataset_attributes.png)
+
+## Analysis with disaggregation categories and attribute categories
+
+In order to do analysis with disaggregation and attribute categories,
+check the “Data dimension” box in the category editing screen of the
+Maintenance app, as discussed in **Chapter 21.2.4: Create or edit a
+category**.
+
+## Approvals with attribute categories
+
+To include attribute categories in approvals, create a category
+option group that contains the same category options as the attribute
+category.  Then create a category option group set and add that the
+category option group set as a data approval level.
+
+This is covered in more detail in **Chapter 16.6: Approving by category
```

Comment:
Elsewhere in this document, top-level numbers are known as "chapters" and anything below are "sections". There are extremely few cross-references to sections in this guide, and those that exist avoid referencing the section number (presumably in case they should change in the future). Two examples of cross-referencing sections that I can find are:

In the Maintenance app section under Data approval level
See the “Analytics settings” section of this user guide.

Following this pattern I suggest:

This is covered in more detail in the Data approval chapter, sections "Approving by category option group set" and "Approving by multiple category option group sets".

Reply:
The image is at:
https://docs.dhis2.org/master/en/user/html/resources/images/content/user/maintainence/categories_dataset_attributes.png

Reply:
@jimgrace, I edited the references to remove the word "Chapter" and made a few other minor changes.  Thanks!